tests/unit/switch_utils.c

```diff
@@ -63,6 +63,24 @@ FST_TEST_BEGIN(benchmark)
 }
 FST_TEST_END()
 
+FST_TEST_BEGIN(b64)
+{
+    char *str = "ABC";
+    unsigned char b64_str[6];
+    char decoded_str[4];
+    switch_status_t status = switch_b64_encode((unsigned char *)str, strlen(str), b64_str, sizeof(b64_str));
+    switch_log_printf(SWITCH_CHANNEL_LOG, SWITCH_LOG_INFO, "b64_str: %s\n", b64_str);
+    fst_check(status == SWITCH_STATUS_SUCCESS);
+    fst_check_string_equals((const char *)b64_str, "QUJD");
+
+    switch_size_t size = switch_b64_decode((const char *)b64_str, decoded_str, sizeof(decoded_str));
+    switch_log_printf(SWITCH_CHANNEL_LOG, SWITCH_LOG_INFO, "decoded_str: %s\n", decoded_str);
+    fst_check_string_equals(decoded_str, str);
+    fst_check(size == 3);
```

Comment:
this should be 4 not 3

Reply:
Hi, Michale

I will do the change now.

Thanks for your comments.

Reply:
Done, Please check.